exercises/D-logical-operators/exercise.js

```diff
@@ -14,11 +14,33 @@ let cssLevel = 4;
 let htmlLevelAbove5;
 let cssLevelAbove5;
 
+if (htmlLevel > 5) {
+  htmlLevelAbove5 = true;
+} else {
+  htmlLevelAbove5 = false;
+}
+
+if (cssLevel > 5) {
+  cssLevelAbove5 = true;
+} else {
+  cssLevelAbove5 = false;
+}
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
 let cssAndHtmlAbove5;
 let cssOrHtmlAbove5;
+if (htmlLevel > 5 && cssLevel > 5) {
+  cssAndHtmlAbove5 = true;
+} else {
+  cssAndHtmlAbove5 = false;
+}
+
+if (htmlLevel > 5 || cssLevel > 5) {
+  cssOrHtmlAbove5 = true;
+} else {
+  cssOrHtmlAbove5 = false;
+}
```

Comment:
Use of conditional (ternary) operator could save typing effort and enable you to assign variables at the same time. This is greatly help you to avoid problem of **undefined** variables